Packs/PrismaCloudCompute/README.md

```diff
@@ -44,12 +44,11 @@ Configure Prisma Cloud Compute to send alerts to Cortex XSIAM by creating an ale
 The integration ships with four default playbooks:
 * **Prisma Cloud Compute - Audit Alert v3**
 * **Prisma Cloud Compute - Cloud Discovery Alert**
-* **Prisma Cloud Compute - Compliance Alert**
+* **Prisma Cloud Compute - Compliance Alert v2**
 * **Prisma Cloud Compute - Vulnerability Alert**
 
-Three of the above playbooks (all except _Audit Alert v3_) contain a single script. The script in each playbook encode the raw JSON alerts into Cortex XSIAM objects that can then be used in the playbooks. The scripts are:
+Two of the above playbooks (all except _Audit Alert v3_) contain a single script. The script in each playbook encode the raw JSON alerts into Cortex XSOAR objects that can then be used in the playbooks. The scripts are:
```

Comment:
Actually it's 2 since therer's another one which doesn't contain a single script - the compliance playbook - which comes right after this one in the readme.

Reply:
I'll make the description clearer.